### ISSUE TITLE
update GNOME runtime to 3.38

### ISSUE THREAD

**Issue by eszlari** [flathub/com.github.calo001.fondo#15]

**Comment by calo001** [comment]
Using this runtime version the wallpaper is not established on the desktop using elementary OS 6, Does anyone have this problem too?

**Comment by JeffLabonte** [comment]
Gnome Runtime 3.36 is not supported since February 2021. It should be updated.

**Comment by HougeLangley** [comment]
Required runtime for com.github.calo001.fondo/x86_64/stable (runtime/org.gnome.Platform/x86_64/3.36) found in remote flathub
Info: org.gnome.Platform//3.36 is end-of-life, with reason:
   The GNOME 3.36 runtime is no longer supported as of February 13, 2021. Please ask your application developer to migrate to a supported platform.

**Comment by hfiguiere** [comment]
tl;dr it won't work, upstream need to fix it.

https://github.com/flathub/com.github.calo001.fondo/pull/21#issuecomment-817068279

**Comment by ghost** [comment]
Closed, due to EOL. https://github.com/flathub/com.github.calo001.fondo/pull/32
